code/index.html

```diff
@@ -5,18 +5,111 @@
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
   </head>
   <body>
     <header>
```

Comment:
Super clean code, and easy to understand. Good job! I'll be cleaning up my code to look more like yours!